azure-pipelines.yml

```diff
@@ -139,6 +139,9 @@ jobs:
     steps:
       - template: ci/report-start.yml
       - checkout: self
+      - bash:
```

Comment:
Maybe a comment that we do this to remove shared memory created by postgres would be useful in the future?